### ISSUE TITLE
[Feature] Adding some simple drawing?

### ISSUE THREAD

**Issue by LamprosPitsillos** [Kl4rry/simp#7]
What are your opinions on some simple drawing features? 
Some shapes , a simple drawing brush with some size and opacity settings?

**Comment by Kl4rry** [comment]
I am not intending to create a drawing program. But I agree that having something like a simple brush to doodle on images would be nice. It does add some complexity as I don't think the code base is super well suited to adding drawing. We would need to create another layer and a another texture for the drawing content to be saved in. There is also some un answered questions like how to handle drawing on animated images like GIFS. Should it modify all frames or should there be a way of drawing on each frame?

TLDR I will probably add it at some point but its not really high prio atm.

**Comment by olivier-fs** [comment]
I'm very happy that simp is a _simple_ _viewer_
It should: 
- open and display image blazingly fast ;-)
- resize/scroll fast
- close fast
- have small memory footprint

I think it should never have  "let's be a drawing tool" as goal : there is plenty of other tools for this.
